docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -5,3 +5,226 @@ noindex: True
+- [Configure an email alert policy](#configuring-an-email-alert-policy)
+
+### Setting an instance deployment's policies
+
+A list of alert policies can be defined in a single YAML file. After declaring your policies, set them for your instance deployment using the following command:
+
+```bash
+dagster-cloud deployment alert-policies sync -a <ALERT_POLICIES_PATH>
+```
+
+### Viewing an instance deployment's policies
+
+List the policies currently configured on your instance deployment by running:
+
+```bash
+dagster-cloud deployment alert-policies list
+```
+
+### Configuring a Slack alert policy
+
+In this example, we'll configure a Slack notification to trigger whenever a run of a job succeeds or fails. This job, named `sales_job`, has a tag of `{"team", "sales"}`:
+
+```python
+@op
+def sales_computation():
+    ...
+
+
+@job(tags={"team": "sales"})
+def sales_job():
+    sales_computation()
+```
+
+In the alert policies YAML file, we'll define a policy that listens for jobs with the `{"team", "sales"}` tag to succeed or fail. When this occurs, a notification will be sent to the `sales-notification` channel in the `hooli` workspace:
+
+```yaml
+alert_policies:
+  - name: "slack-alert-policy"
+    description: "An alert policy to send a Slack notification to sales on job failure or success."
+    tags:
+      - key: "team"
+        value: "sales"
+    event_types:
+      - "JOB_SUCCESS"
+      - "JOB_FAILURE"
+    notification_service:
+      slack:
+        slack_workspace_name: "hooli"
+        slack_channel_name: "sales-notifications"
+```
+
+### Configuring an email alert policy
+
+In this example, we'll configure an email alert when a job fails. This job, named `important_job`, has a tag of `{"level": "critical"}`:
+
+```python
+def important_computation():
+    ...
+
+
+@job(tags={"level": "critical"})
+def important_job():
+    important_computation()
+```
+
+In the alert policies YAML file, we'll define a policy that listens for jobs with the `{"level": "critical"}` tag to fail. When this occurs, an email notification will be sent to `richard.hendricks@hooli.com` and `nelson.bighetti@hooli.com`:
+
+```yaml
+alert_policies:
+  - name: "email-alert-policy"
+    description: "An alert policy to email company executives during job failure."
+    tags:
+      - key: "level"
+        value: "critical"
+    event_types:
+      - "JOB_FAILURE"
+    notification_service:
+      email:
+        email_addresses:
+          - "richard.hendricks@hooli.com"
+          - "nelson.bighetti@hooli.com"
+```
+
+---
+
+## Using system tags to configure alert emails
+
+For a job, alert emails can be configured by setting the `dagster-cloud/alert_emails` tag on a job. When a job run fails, a notification will be sent to the alert emails.
+
+In this example, we've defined two alert emails for the `important_job` job: `richard.hendricks@hooli.com` and `nelson.bighetti@hooli.com`. On run failure, these two emails will be sent a notification:
```

Comment:
This is the old legacy thing that alert policies replaced, not sure if we want to make that clear in the docs